### ISSUE TITLE
Map list scrolls to Shared with me section instead of top (On device) on refresh. 

### ISSUE THREAD

**Issue by jdzurisin** [doug-m/CollectorAuroraTesting#229]
### Issue

Map list scrolls to Shared with me section instead of top (On device) on refresh. 
### Expected Behavior

Seems to me that maps that the app should scroll either to On Device or Recent maps instead as groups that have a large number of maps cause issues with UX.
### Map and Credentials

* Group Name:
* Map Name:

### Device Information
* Device: MotoG7Power MotoG5Plus
* OS Version: 9, 6

**Comment by Kevin-GIS** [comment]
This is a known issue - Apps-Android 968

**Comment by dmcilwr** [comment]
Agreed - I would also expect the app to take me to On Device or Recent, Just seems counter intuitive that it is taking me to the bottom of the list of items instead of the top (regardless of what section is the top).
Related note - find the refresh to be super sensitive when trying to scroll to the top when using the account with a lot of maps shown - before user can lift their finger off the screen the app is already re-freshing and dropping them back down the list.
